src/main/scala/piconot/external/RightHand.bot

```diff
@@ -0,0 +1,13 @@
+# jk, this doesnt work
+# we don't support super multiple nesting
+# also, this is too hard without knowing which direction the bot is facing
+
+while (any) {
```

Comment:
Multiple nesting definitely seems hard! However, could you have gotten around multiple nesting if you revisited your idea of using &&? That way, if S is free and E is free and another letter is free, you don't have to nest them within each other, you can just do a while loop for when S is free, a separate while loop for when S&E are free, etc. Not sure if this could have helped in your right hand program.